index.js

```diff
@@ -211,14 +210,14 @@ RedisClient.prototype.do_auth = function () {
     self.send_anyway = true;
     self.send_command("auth", [this.auth_pass], function (err, res) {
         if (err) {
-            if (err.toString().match("LOADING")) {
+            if (/LOADING/.test(err.toString())) {
```

Comment:
I wouldn't worry too much about perf here because it's in the `auth` stanza, which is only used by a subset of Redis users, and even then is only called once per client connection, but not only that it's in the error pathway of the auth stanza, so this is a super rare code path.

I shouldn't have brought perf up ealier :grin: 

Mostly just need to worry about correctness